PopupBridge/build.gradle

```diff
@@ -22,7 +22,7 @@ android {
 }
 
 dependencies {
-    api 'com.braintreepayments:browser-switch:1.1.2'
+    implementation 'com.braintreepayments.api:browser-switch:2.0.0-beta1'
```

Comment:
Maybe we should add this to the changelog as well? I assume we'll use a non-beta version when we actually release pop-up bridge, but we could add a changelog entry now so that we don't forget later.

Reply:
Good idea - added that entry